ledger/ledger-api-tests/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_8/UserManagementServiceInvalidUpdateRequestTests.scala

```diff
@@ -0,0 +1,181 @@
+          updatePaths = Seq.empty,
+        )
+      )
+      .mustFailWith(
+        "update with an unknown update path",
+        errorCode = LedgerApiErrors.Admin.UserManagement.InvalidUpdateUserRequest,
+        exceptionMessageSubstring = Some(
+          s"INVALID_ARGUMENT: INVALID_USER_UPDATE_REQUEST(8,0): Update operation for user id '${user.id}' failed due to: The update mask contains no entries"
+        ),
+      )
+  })
+
+  userManagementTestWithFreshUser(
+    "InvalidUpdateRequestsInvalidUserIdSyntax",
+    "Failing update requests when user id is not a valid user id",
+  )()(implicit ec => { (ledger, _) =>
+    ledger.userManagement
+      .updateUser(
+        updateRequest(
+          id = "%%!!!",
+          annotations = Map("k2" -> "v2"),
+          updatePaths = Seq.empty,
+        )
+      )
+      .mustFailWith(
+        "update with an unknown update path",
+        errorCode = LedgerApiErrors.RequestValidation.InvalidField,
+        exceptionMessageSubstring = Some(
+          s"INVALID_ARGUMENT: INVALID_FIELD(8,0): The submitted command has a field with invalid value: Invalid field user.id: User ID \"%%!!!\" does not match regex \"[a-z0-9@^$$.!`\\-#+'~_|:]{1,128}\""
+        ),
+      )
+  })
+
+  userManagementTestWithFreshUser(
+    "InvalidUpdateRequestsUpdatingUnmodifiableField",
+    "Failing update requests when attempting to update an unmodifiable field",
+  )()(implicit ec => { (ledger, user) =>
+    ledger.userManagement
+      .updateUser(
+        updateRequest(
+          id = user.id,
+          updatePaths = Seq("user.id"),
+        )
+      )
+      .mustFailWith(
+        "update with an unknown update path",
+        errorCode = LedgerApiErrors.Admin.UserManagement.InvalidUpdateUserRequest,
+        exceptionMessageSubstring = Some(
+          s"INVALID_ARGUMENT: INVALID_USER_UPDATE_REQUEST(8,0): Update operation for user id '${user.id}' failed due to: The update path: 'user.id' points to an unknown or an unmodifiable field."
+        ),
+      )
+  })
+
+  userManagementTestWithFreshUser(
+    "InvalidUpdateRequestsEmptyUpdatePath",
+    "Failing update requests when the update mask contains an empty update path",
+  )()(implicit ec => { (ledger, user) =>
+    ledger.userManagement
+      .updateUser(
+        updateRequest(
+          id = user.id,
+          annotations = Map("k2" -> "v2"),
+          updatePaths = Seq(""),
+        )
+      )
+      .mustFailWith(
+        "update with an unknown update path",
+        errorCode = LedgerApiErrors.Admin.UserManagement.InvalidUpdateUserRequest,
+        exceptionMessageSubstring = Some(
+          s"INVALID_ARGUMENT: INVALID_USER_UPDATE_REQUEST(8,0): Update operation for user id '${user.id}' failed due to: The update path: '' has invalid syntax."
+        ),
+      )
+  })
+
+  userManagementTestWithFreshUser(
+    "InvalidUpdateRequestsUserFieldIsUnset",
+    "Failing update requests when user field is unset",
```

Comment:
Wasn't there also an error when the path is double-specified?

Reply:
There is one. Added a test case for doubly specified paths